### DISCUSSION TITLE
Feature Request: Auto-generate field labels if none specified

### DISCUSSION THREAD

**Discussion by jmikrut** [payloadcms/payload#123] [Feature Requests & Ideas]
# Feature Request

Auto-generate field labels based on the field's `name`.  Allow for labels to be disabled by passing `false` to the `label` field property.

## Use-Case

It gets a bit tedious to specify field labels over and over, when the `name` of the field could probably be used and formatted properly to generate a field label automatically.

By generating field labels automatically during field sanitization, there will be less configuration necessary by the developer.

## Additional Detail

We need to make sure to adjust the field schema validation _**and**_ field type(s) to support either strings or booleans to be passed to the `label` property of all fields that support labels.

We also need to make sure to update Field documentation with the corresponding API changes.

**Comment by denolfe** [comment]
PR on #42
